pkg/cloudevents/transport/nats/codec_test.go

```diff
@@ -237,7 +237,7 @@ func TestCodecRoundTrip(t *testing.T) {
 					got.Data = data
 				}
 
-				if tc.wantErr != nil || err != nil {
```

Comment:
This does not catch the case where wantErr is nil and err is not.

Reply:
See https://github.com/cloudevents/sdk-go/pull/163/files/1807d618dcaa5673fd594d50c78e8a89b7fd3039#r306909479